lib/plottr_components/src/components/dashboard/files/FilesHome.js

```diff
@@ -37,6 +45,21 @@ const FilesHomeConnector = (connector) => {
     }
   }
 
+  function scrivenerImportDialog() {
+    const title = t('Choose your Scrivener project')
+    const filters = [{ name: t('Scrivener file'), extensions: ['scriv'] }]
+    const properties = ['openFile', 'createDirectory']
+    const files = showOpenDialogSync({ title, filters, properties })
+    if (files && files.length) {
+      const importedPath = files[0]
+      if (importedPath.toLowerCase().includes('.scriv')) {
+        return importedPath
+      } else {
+        return null
```

Comment:
Perhaps we should tell the user that they made a mistake?
You could add another case to the `error` reducer.